.github/workflows/tag-and-release.yaml

```
@@ -1,5 +1,7 @@
 name: Tag and Release
 on:
+  push:
+    - '*'
```

actionlint

"push" section is sequence node but mapping node is expected

(syntax-check)